### PR TITLE
DDCE-1751: upgraded bootstrap version and upgraded to play 2.7

### PR DESCRIPTION
**Pull request by fraser-sprott**

### CODE REVIEW

File: app/uk/gov/hmrc/homeofficesettledstatus/filters/AuditFilter.scala [hunk 1]
```diff
@@ -17,22 +17,25 @@
 package uk.gov.hmrc.homeofficesettledstatus.filters
 
 import akka.stream.Materializer
-import javax.inject.Inject
+import play.api.Configuration
+
+import javax.inject.{Inject, Singleton}
```
Comment (anburoach) [inline app/uk/gov/hmrc/homeofficesettledstatus/filters/AuditFilter.scala:22]: Plz remove unused import

File: app/uk/gov/hmrc/homeofficesettledstatus/filters/HossFilters.scala [hunk 2]
```diff
@@ -14,16 +14,32 @@
...
-import uk.gov.hmrc.play.bootstrap.filters.FrontendFilters
-
-class HossFilters @Inject()(
-  shutteringFilter: ShutteringFilter,
-  frontendFilters: FrontendFilters,
-  auditFilter: AuditFilter
-) extends DefaultHttpFilters(
-      (shutteringFilter +: frontendFilters.filters
-        .filterNot(_.getClass.getSimpleName.contains("AuditFilter")) :+ auditFilter): _*)
+///*
```
Comment (anburoach) [inline app/uk/gov/hmrc/homeofficesettledstatus/filters/HossFilters.scala:17]: I see all lines commented in this file.. please remove this file if it is not required

File: it/uk/gov/hmrc/homeofficesettledstatus/stubs/JourneyTestData.scala [hunk 3]
```diff
@@ -4,6 +4,7 @@ import java.time.LocalDate
 
 import uk.gov.hmrc.domain.Nino
 import uk.gov.hmrc.homeofficesettledstatus.models.{ImmigrationStatus, StatusCheckByNinoRequest, StatusCheckResult}
+import java.time.LocalDate
```
Comment (anburoach) [inline it/uk/gov/hmrc/homeofficesettledstatus/stubs/JourneyTestData.scala:7]: remove duplicate import

File: project/plugins.sbt [hunk 4]
```diff
@@ -3,18 +3,20 @@ resolvers += Resolver.url(
...
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.5")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.0.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.6.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.13.0")
```
Comment (anburoach) [inline project/plugins.sbt:14]: sbt-artifactory is not required any more, so you can remove it

File: test/uk/gov/hmrc/homeofficesettledstatus/views/ViewsSpec.scala [hunk 5]
```diff
@@ -26,13 +26,13 @@ import play.api.test.Helpers._
 import play.twirl.api.Html
 import uk.gov.hmrc.homeofficesettledstatus.views.html.{HossFooterLinks, error_template, govuk_wrapper, main_template}
 import uk.gov.hmrc.play.config.{AssetsConfig, GTMConfig, OptimizelyConfig}
```
Comment (anburoach) [inline test/uk/gov/hmrc/homeofficesettledstatus/views/ViewsSpec.scala:28]: Remove unused import

File: project/plugins.sbt (hunk 4 above)
Comment (fraser-sprott) [reply project/plugins.sbt:14]: not 100% sure on this one, is this documented anywher. Not seen anything about this

Comment (fraser-sprott) [reply project/plugins.sbt:14]: just found he post about it

**Review by anburoach — APPROVED** [review]
Approved

**Pull request merged**